src/main/java/de/unistuttgart/towercrushbackend/controller/websockets/WebsocketController.java

```diff
@@ -12,73 +12,34 @@
 import org.springframework.messaging.simp.SimpMessagingTemplate;
 import org.springframework.stereotype.Controller;
 
-import java.time.LocalDateTime;
-import java.util.HashMap;
-import java.util.Map;
-import java.util.concurrent.ExecutorService;
-import java.util.concurrent.Executors;
-import java.util.concurrent.Future;
 
 @Controller
 @Slf4j
 public class WebsocketController {
 
     @Autowired
     SimpMessagingTemplate simpMessagingTemplate;
-    String destination = "/topic/messages";
-    String gameDestination = "/topic/games/";
-
-    ExecutorService executorService =
-        Executors.newFixedThreadPool(3);
-    Map<String, Future<?>> submittedLobbies = new HashMap<>();
-    @MessageMapping("/start/lobbyName/{lobbyName}")
-    public void startGame(@DestinationVariable String lobbyName) {
-        log.info("Started task");
-        MessageWrapper startMessage = new MessageWrapper("Started game: " + lobbyName, Purpose.CHAT_MESSAGE);
-        simpMessagingTemplate.convertAndSend(gameDestination + lobbyName,
-            startMessage);
-        submittedLobbies.put(lobbyName, executorService.submit(() -> {
-            while (true) {
-                MessageWrapper idleMessage = new MessageWrapper(LocalDateTime.now()
-                    + ": doing game: " + lobbyName, Purpose.CHAT_MESSAGE);
-                simpMessagingTemplate.convertAndSend(gameDestination + lobbyName,
-                    idleMessage);
-                Thread.sleep(10000);
-            }
-        }));
+    final String destination = "/topic/messages";
+    final String lobbyDestination = "/topic/lobbies/";
```

Comment:
is that a URL?
If yes
```suggestion
    final String lobbyEventURL = "/topic/lobbies/";
```
assuming I understand its purpose correctly…

Reply:
well it just defines to which toppic you want to connect, so its kinda the destination prefix, i dont like your name as much as the default one.